node/churner_client.go

```diff
@@ -45,8 +46,12 @@ func (c *churnerClient) Churn(ctx context.Context, operatorAddress string, keyPa
 		return nil, errors.New("quorumIDs cannot be empty")
 	}
 	// generate salt
-	privateKeyBytes := []byte(keyPair.PrivKey.String())
-	salt := crypto.Keccak256([]byte("churn"), []byte(time.Now().String()), quorumIDs[:], privateKeyBytes)
+	bytes := make([]byte, 32)
+	_, err := rand.Read(bytes)
+	if err != nil {
+		return nil, err
+	}
+	salt := crypto.Keccak256([]byte("churn"), []byte(time.Now().String()), quorumIDs[:], bytes)
```

Comment:
perhaps, we can set bytes = salts. Not sure why do keccak256().

But if leave as what it is, the code is still correct

Reply:
@gpsanant might know why we do it?